custom-metrics-autoscaling/prometheus-to-sd/prometheus_dummy_exporter.go

```diff
@@ -0,0 +1,50 @@
+/*
+Copyright 2018 The Kubernetes Authors.
```

Comment:
these should be `2018 Google Inc`

also add license header to Dockerfiles and yamls.

Reply:
Other Dockerfiles and yamls in this repo don't have any license headers, so I don't have an example for those. Should it be just `// Copyright 2018 Google Inc`?

Reply:
that's likely a historic artifact that predates me.

What's worrying here is that you’re bringing a copyrighted file. Did they come from CNCF/k8s repos?

Reply:
No, they came from https://github.com/GoogleCloudPlatform/k8s-stackdriver/ repo.

Reply:
Fixed headers in all files (except READMEs)